ui-v2/app/components/consul-token-list/pageobject.js

```diff
@@ -1,6 +1,6 @@
 export default (collection, clickable, attribute, text, actions) => () => {
-  return collection('.consul-token-list li:not(:first-child)', {
-    id: attribute('data-test-token', '[data-test-token]'),
+  return collection('.consul-token-list [data-test-list-row]', {
+    id: attribute('data-test-token', 'a[data-test-token]'),
```

Comment:
I think you can remove this `a`, that way ever need to move the `data-test-*` attribute to another node we don't have to change this (unlikely we'll have to move nodes, but the point of these selectors is so they are independent of the nodeNames/classes etc themselves)